src/templates/init/jest.integration-test.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  transform: {"^.+\\.tsx?$": "ts-jest"},
+  roots: ["integration-tests"],
+  reporters: [
+    'default',
+    [ "jest-junit", { suiteName: "jest tests",outputDirectory:"s4hana_pipeline/reports/backend-integration/" } ]
+  ]
```

Comment:
Formatting suggestion
``` ts
[
  "jest-junit",
  {
    suiteName: "jest tests",
    outputDirectory: "s4hana_pipeline/reports/backend-integration/" 
  }
]
```

Reply:
hm, The suiteName is optinal and I removved it. It would prefe a single line